05_syntax_lab/03.py

```diff
@@ -1,7 +1,7 @@
-"""
-Write a program that randomizes a number
-and prints the sum total of its digits.
-For example if the number was: 2345
-The result should be: 14
-"""
-
+from random import randint
+number=randint(1,10000)
+sum=0
+for i in str(number):
+    sum+=int(i)
+print number
+print sum
```

Comment:
👍 nice, but please leave a space around operators: ```sum += int(i)``` to make the code more readable.